### DISCUSSION TITLE
PCL远古屎山复现现场

### DISCUSSION THREAD

**Discussion by wuliaodexiaoluo** [Meloong-Git/PCL#3416] [讨论]
![图片](https://github.com/Hex-Dragon/PCL2/assets/160083863/da760c7f-a872-4fca-a401-ed20556acd74)

**Comment by wuliaodexiaoluo** [comment]
这个搜索Bug还存在于2.2.14
远古屎山是吧

**Comment by allMagicNB** [comment]
一开始是没有限制的，但是有你这种故意找茬的人，便有了限制

**Comment by allMagicNB** [comment]
~~话说你咋还没当上 Collaborator~~

**Comment by ghost** [comment]
Test

**Reply by allMagicNB** [reply]
哟，大号复活了（

**Comment by ghost** [comment]
> ~话说你咋还没当上 Collaborator~

我也很好奇为什么。龙猫再不设就只能等下个假期了。。。

**Reply by allMagicNB** [reply]
~~请，好好地看着我的右边的某个圆角方框~~

**Comment by wuliaodexiaoluo** [comment]
龙猫大概忘了我叭
上次就是忘了6月份的，然后我的赞助者榜单一直等到12月才有。。。

**Comment by MoYuan-CN** [comment]
版聊呢....

**Reply by allMagicNB** [reply]
这里是 Discussion，版聊没啥关系（

**Comment by wuliaodexiaoluo** [comment]
> ~请，好好地看着我的右边的某个圆角方框~

你再炫小心龙猫撤了

**Comment by wuliaodexiaoluo** [comment]
猫曰：没有Author，Collaborator没有任何意义

**Comment by wuliaodexiaoluo** [comment]
我甚至复现了版本被追加收藏的Bug，证明了这是个远古屎山

**Reply by allMagicNB** [reply]
> 我甚至复现了版本被追加收藏的Bug，证明了这是个远古屎山

这和屎山没多大关系……

**Comment by allMagicNB** [comment]
首先，你回复别人的时候回帖就行了，多发一个帖有点乱

**Comment by kong-que-dnf123** [comment]
所以到底是啥bug存在很久？
CPU异常占用率吗？？？

**Reply by wuliaodexiaoluo** [reply]
对，还有追加收藏的Bug

**Comment by Ad-closeNN** [comment]
这怎么还有一个这里找不到的人
是不是有个comment被删了（
![image](https://github.com/Hex-Dragon/PCL2/assets/103355026/b41da7db-0588-4078-84fa-ea7971c26734)

**Reply by wuliaodexiaoluo** [reply]
被某个sb Collaborator hide了

**Reply by Ad-closeNN** [reply]
看到了
![image](https://github.com/Hex-Dragon/PCL2/assets/103355026/1d233f90-0dc9-4df6-be53-bc871e9c44a8)

**Reply by Ad-closeNN** [reply]
这专为 Collaborator 和 Member 设计的（虽然我不是这两个的任何一个）watch还挺好用（
![image](https://github.com/Hex-Dragon/PCL2/assets/103355026/a46dd67b-0b1d-44fe-8c45-a453019275f1)

**Comment by MoYuan-CN** [comment]
请友好交流
这里是 Discussion，不要开一堆楼，请在已有楼下方使用 Write a reply 功能
不要版聊，不然订阅了这个仓库的那些人邮箱该炸了...

**Reply by wuliaodexiaoluo** [reply]
~You can ask them to touch the button "Unsubscribe"~

**Reply by allMagicNB** [reply]
> ~You can ask them to touch the button "Unsubscribe"~

我不能，所以我可以锁了吧？
![Screenshot_20240223_190009.jpg](https://github.com/Hex-Dragon/PCL2/assets/116283787/880abdff-ebff-41d7-8ae9-ea9998db2050)
**不正常的可以锁帖或删帖**

**Reply by wuliaodexiaoluo** [reply]
~滥用职权是吧。马上给你举报喽~
我恶意了吗，你凭什么认为我恶意辱骂你，你是我吗 ~(子非鱼，安知鱼之乐也)~

**Reply by Deep-Dark-Forest** [reply]
不对吧，无论怎么聊怎么回复都会炸邮箱
